src/utils.ts

```diff
@@ -37,14 +37,14 @@ export async function loadTemplateRegistry(): Promise<TemplateRegistry> {
   }
 }
 
-const versionRegex = /@(v[0-9][^\/]*)\//gm;
-
 function calulateVersions(registry: TemplateRegistry) {
   for (const tmpl of Object.values(registry.templates)) {
     if (tmpl.version) {
       continue;
     }
 
+    const versionRegex = /@(v[0-9][^\/]*)\//g;
```

Comment:
is this meant to be a loose match on version strings? This regex would match `@v0-foo-bar-😄-🤷‍♂️`

Reply:
I didn't want it to strict but we certainly _don't_ want Unicode characters :)

Updated
```
@(v[0-9][.0-9a-zA-Z\-_^\/]*)\/
```